opba-db/src/main/java/de/adorsys/opba/db/domain/entity/Consent.java

```diff
@@ -75,6 +75,9 @@
     @LastModifiedDate
     private Instant modifiedAt;
 
+    @Column(name = "image_data")
```

Comment:
This data must be in context, not in payment

Reply:
Removed